src/services/multiplayerGameService.js

```diff
@@ -0,0 +1,102 @@
+import config from '../../config';
+
+const DEFAULT_HEADER_JSON = {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+};
+
+const DEFAULT_BASE_URL = config.apiUrl;
+
+function handleErrors(res) {
+    if (!res.ok) {
+        return Promise.reject(res.statusText);
+    }
+    return res;
+}
+
+export const newGame = (baseUrl = DEFAULT_BASE_URL) => (mode = 'multi') => {
+    const url = `${baseUrl}/game`;
```

Comment:
You should use url, method and headers directly in fetch call instead of declare variables